backend/spec/features/admin/configuration/general_settings_spec.rb

```diff
@@ -39,5 +39,31 @@
 
       expect(page).to have_content(Spree.t(:clear_cache_ok))
     end
+
+    context "with Spree mounted to a custom path" do
```

Comment:
Okay, I'll remove the spec. (Darn, and after I spent a bunch of time figuring out how to do this! :smile: )